deploy/export-url-bucket-cors.json

```diff
@@ -4,8 +4,8 @@
          "GET"
       ],
       "origin":[
-         "https://bvdp-saturn-prod.appspot.com",
-         "https://bvdp-saturn-dev.appspot.com"
+         "https://app.terra.bio",
+         "https://app.terra.bio"
```

Comment:
Leave https://bvdp-saturn-dev.appspot.com, I think they're still using that for dev